src/styles/_header.scss

```diff
@@ -37,5 +36,12 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color ease;
+    transition-duration: $t-duration;
```

Comment:
you are already using shorthand prop. squeeze the duration in there also

Reply:
Fixed